src/Views/ThirdPartySoftwareView.vala

```diff
@@ -0,0 +1,82 @@
+/*-
```

Comment:
This file is now duplicated by SoftwareView.vala.

Reply:
Strange 🤔 I thought I renamed the file.